runner/browserrunner.ts

```diff
@@ -203,7 +203,7 @@ export class BrowserRunner {
 
     // Nothing to quit.
     if (!this.sessionId) {
-      return this.doneCallback(error, this);
+      error ? this._reject(error) : this._resolve();
```

Comment:
This is true. Wanted: a convenient, legible, and common way of converting a node callback into a promise. This ternary saves 5 lines, and its intent seems clear to me but I'm probably too close to it to call it.